readme.md

```diff
@@ -1,11 +1,11 @@
 # Tachyons Documentation
 
-Documentation for Tachyons -- a performant, mobile-first, and 100% responsive modular css framework.
+Documentation for Tachyons — a performant, mobile-first, and 100% responsive modular CSS framework.
 
 ## Installation and Usage
 
 ```
-git clone https://github.com/tachyons-css/tachyons-css.github.io && cd $_
```

Comment:
I tried simply putting this `cd` command on the next line, but the directory is not called `https://github.com/tachyons-css/tachyons-css.github.io`.